deploy/cromwell/kubernetes/nginx.conf

```diff
@@ -0,0 +1,36 @@
+# nginx server for static assets. For directive documentation, see
+# http://nginx.org/en/docs/dirindex.html
+
+# Required - just leave the defaults for now.
+events {}
+
+http {
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
+  gzip on;
+  gzip_disable "msie6";
+
+  server {
+    listen 80;
+
+    root /ui/dist;
+
+    location / {
+      try_files $uri$args $uri$args/ /index.html;
+    }
+
+    location /api {
+      proxy_pass http://127.0.0.1:8190;
+    }
+
+    # This is used for health check, which is required by Kubernetes
+    location /health {
+      return 200 "healthy!\n";
+      # because default content-type is application/octet-stream,
+      # browser will by default try to save the file
+      # below line allows you to see it in the web page
+      add_header Content-Type text/plain;
+    }
+  }
+}
```

Comment:
tiny nit: new line

Reply:
+1

Reply:
(ps if you use intellij or pycharm to make these: https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file )